sdk_container/src/third_party/coreos-overlay/coreos-base/afterburn/files/coreos-metadata.service

```diff
@@ -21,6 +21,8 @@ ConditionKernelCommandLine=|flatcar.oem.id=scaleway
 
```

Comment:
Besides the metadata afterburn also supports the hostname and the ssh keys for this provider. We should add this in `bootengine` (`flatcar-metadata-hostname.service`) and `init` (`sshkeys.service`)

Reply:
I added this in the case of fallback, but this image should be used with Ignition userdata format. The default flatcar production image with no oem works in the case of openstack metadata/userdata format on KubeVirt, as Kubevirt is just a config drive implementation of OpenStack metadata.

Reply:
About coroes-cloudinit: The mount unit trigger will only run when Ignition didn't run. There is no explicit enablement because there is no `--oem kubevirt` flag for coreos-cloudinit.